src/components/Header/index.js

```diff
@@ -7,9 +7,28 @@ class Header extends React.Component {
 
   constructor(props) {
     super(props);
+
+    this.state = {
+      embed: false
+    };
+  }
+
+  componentDidMount() {
+    !this.state.embed ? this.isEmbed() : '';
+  }
+
+  isEmbed() {
```

Comment:
Let's simplify this method to just be:
```javascript
checkIfEmbed() { /* With the previous comment */
  const isEmbed = /embed=true/g.test(window.location.hash);
  if(isEmbed) {
    this.setState({ embed: true });
  }
}
```

Anyway, I don't understand why you use `window.location.hash` whereas we already have the router params stored into the App's state.